### PR TITLE
Cleanup and renaming

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

File: docs/source/contributing/volto-core-addons.md [hunk 1]
```diff
@@ -3,18 +3,18 @@
 Volto has the concept of "core add-ons".
 They are add-ons that are always installed in Volto.
 They are located separately from the main code in the `packages` folder.
-These add-ons can be released as standalone add-ons, and used in projects that are using versions of Volto that do not have them by default.
-Another useful use case is the introduction of new or experimental features that need to be developed side-by-side with core Volto, but not enabled by default.
+If Volto uses them directly, these add-ons must be released as standalone add-ons.
```
Comment (stevepiercy) [inline docs/source/contributing/volto-core-addons.md:6]:
```suggestion
If Volto uses them directly, these add-ons must be released as stand-alone add-ons.
```

File: docs/source/contributing/volto-core-addons.md [hunk 2]
```diff
@@ -3,18 +3,18 @@
...
 You can use these packages released as standalone add-ons installed in your old Volto projects to test them or to have a bleeding edge feature without having to upgrade to the latest Volto version.
+You can add them to non-Volto projects in order to get a piece of Volto feature set outside the main project.
```
Comment (stevepiercy) [inline docs/source/contributing/volto-core-addons.md:8]:
```suggestion
You can add them to non-Volto projects to get parts of the Volto feature set outside of the main project.
```

File: packages/registry/news/5456.internal [hunk 3]
```diff
@@ -0,0 +1 @@
+Cleanup Registry and rename `packagesFolderAddons` to `coreAddons` @sneridagh
```
Comment (stevepiercy) [inline packages/registry/news/5456.internal:1]:
```suggestion
Cleaned up Registry and renamed `packagesFolderAddons` to `coreAddons`. @sneridagh
```

File: packages/volto/news/5464.internal [hunk 4]
```diff
@@ -0,0 +1 @@
+Cleanup Registry and rename `packagesFolderAddons` to `coreAddons` @sneridagh
```
Comment (stevepiercy) [inline packages/volto/news/5464.internal:1]:
```suggestion
Cleaned up Registry and renamed `packagesFolderAddons` to `coreAddons`. @sneridagh
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Minor grammar fixes.

**Review by stevepiercy — APPROVED** [review]
Docs only approved. Thank you!
